### ISSUE TITLE
Accessibility Audit for Drupal 9.5/10 Modules

### ISSUE THREAD

**Issue by edmund-dunn** [department-of-veterans-affairs/va.gov-cms#10628]
**1. What is this request for?**

> Examples: A new product; an update to an existing product.

**2. What is the primary function and/or purpose of the product?**

> Examples: To complete the form in order to contact the VA, to find a facility and look up hours and services at that facility, to simply display content.

**3. Are there any known accessibility issues that have not yet been addressed? If so, please add GitHub links to those issues below.**
  
**4. Are there any specific testing requirements that should be noted?**

> Examples: IE is no longer supported by default but the product does need to be reviewed on IE; the product is not intended to be displayed on mobile.
  
**5. Please provide anything needed to access and review the product.**
  
> Examples: URL(s) for the environment where the product is available for review; Login if needed; Wireframes; Design mockups, etc.
  
**6. When does the product need to launch?**

> All requests should be submitted at least 1 sprint in advance so that it can be worked into the next sprint.  For requests needed sooner, please notify Laura Flannery in the cms-support channel.

**Comment by edmund-dunn** [comment]
@laflannery this can languish in the backlog until the Drupal 10 upgrade is done. It is just serving as a place holder.

**Comment by EWashb** [comment]
FYI @laflannery we will prob start prepping for Drupal 10 in Q1 2023. Possibly late in that quarter.

**Comment by laflannery** [comment]
Now that we have decided to move to 9.5 first this will probably need to be done there first.  Whether it will need to be done again once we move to 10 is not clear yet.

**Comment by EWashb** [comment]
@laflannery do you think we are at a good point to start this?

**Comment by laflannery** [comment]
@EWashb I know the previous comment says we weren't quite sure about 9.5 or 10 but I think this is more towards a 10 audit so it would happen once Edmund's environment is ready for testing. But we can confirm with him and see if he concurs

**Comment by laflannery** [comment]
Drupal 10 instance: https://vacms-12252-drupal-10-cne7edu0najqzzmwjbq3awbbjcxb40w3.ci.cms.va.gov/
Epic to add issues to: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11614

**Comment by EWashb** [comment]
@laflannery you already completed the audit? Or is there still more to do here?

**Comment by laflannery** [comment]
@EWashb I did a good amount of it but didn't get a chance to finish yet

**Comment by laflannery** [comment]
Issues added to Edmund's D10 epic (noted above)
